### ISSUE TITLE
New Tooltips for Page 1

### ISSUE THREAD

**Issue by Biuwa** [hackforla/tdm-calculator#1231]
@Reneedesigns Please mockup the field items having their existing tooltips and new ones for these items
- Project Name
  - The project name should be the way that you refer to the project in conversation
- Building Permit #
 - If you already have a Building Permit # and can provide it here, it may help reduce staff time when reviewing your materials.
- LADOT Case #
  - If you already have a LADOT Case # and can provide it here, it may help reduce staff time when reviewing your materials.
- City Planning Case #
-  If you already have a Planning Case # and can provide it here, it may help reduce staff time when reviewing your materials.
- Project Description
  - As described in your Planning or Building Permit application materials Planning or Building Permit application materials

Once this clickable Figma Prototype is complete, work with Research to test if the following hypothesis is true: users who are exposed to the new page 1, already know how to find tooltips on page 2 (this should be a new issue, once your mockup is made).

- #1218 has been created to get the final language signoff from LACP/LADOT
- Has a planned tooltip already Project Description #1158

_Originally posted by @ExperimentsInHonesty in https://github.com/hackforla/tdm-calculator/issues/1179#issuecomment-1186245286_
